docker/drama-free-django/_test.sh

```diff
@@ -16,15 +17,11 @@ if [ ! -d "$artifact_volume" ]; then
     exit 1
 fi
 
-# Install runtime requirements.
-yum install -y centos-release-scl
-yum install -y python27
-
-source /opt/rh/python27/enable
 
 # Extract the artifact in /tmp.
-cp "$artifact_volume/$artifact_filename" /tmp
-cd /tmp
+mkdir -p $dfd_test_dir
```

Comment:
Not a big deal, but I'm curious why you made a new directory for this. I figured that working in /tmp was fine since the container is ephemeral.

Reply:
The issue was that by having it in `/tmp`, it resulted in directories being created in the `/` root directory, and the non-`root` user didn't have permission to create those directories.  An alternative would be to open up permissions on `/`, but this seems like a better option.